satpy/tests/reader_tests/test_fci_l1c_fdhsi.py

```diff
@@ -426,13 +419,11 @@ def test_handling_bad_data_ir(self):
             "20170410113925_20170410113934_N__C_0070_0067.nc",
         ]
 
-        reader = load_reader(self.reader_configs)
+        reader = load_reader(reader_configs)
         loadables = reader.select_files_from_pathnames(filenames)
         reader.create_filehandlers(loadables)
-        with self.assertLogs(
-                'satpy.readers.fci_l1c_fdhsi',
-                level="ERROR") as cm:
+        with caplog.at_level("ERROR"):
             reader.load([DatasetID(
                     name="ir_123",
                     calibration="brightness_temperature")])
-        self.assertRegex(cm.output[0], "cannot produce brightness temperatur")
+            assert "cannot produce brightness temperatur" in caplog.text
```

Comment:
```suggestion
            assert "cannot produce brightness temperature" in caplog.text
```

Reply:
That typo was already there before :)

Reply:
I think @ameraner is fixing it in his PR

Reply:
Yes I do! See https://github.com/pytroll/satpy/pull/1135/files#diff-78bf1063f7ffb6655291b1b2aa2c06a0R466

Reply:
We're likely to see merge conflicts between those PRs.